src/main.js

```diff
@@ -18,7 +18,17 @@ menuCloseButton.addEventListener('click', closeMenu);
 saveIdeaButton.addEventListener('click', saveIdea);
 userNewTitle.addEventListener('input', verifyForm);
 userNewBody.addEventListener('input', verifyForm);
-starredIdeaButton.addEventListener('click', displayStarredIdeas);
+var timesClicked = 0;
+starredIdeaButton.addEventListener('click', function() {
+  timesClicked++;
+  if(timesClicked%2==0) {
+    starredIdeaButton.innerText = "Show Starred Ideas";
+    showUsersIdeaCard();
+  } else {
+    starredIdeaButton.innerText = "Show All Ideas";
+    displayStarredIdeas();
+  }
+});
```

Comment:
created a timesClicked variable, incremented it based off how many times it's clicked. so it's clicked an even # of times, showUsersIdeaCard() runs, otherwise the odd # of times show displayStarredIdeas()

Reply:
Very cool solution.